packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -135,21 +135,22 @@ The ChatAdapter and the ChatComposite are purpose-built to support single chat t
+| **`offStateChange`**                 | Unsubscribes the handler to stateChanged events.                                                                  | `(handler: (state: `[`ChatAdapterState`](#chatadapterstate)`) => void) => void`  |
+| **`getState`**                       | Get the current State                                                                                             | `() => `[`ChatAdapterState`](#chatadapterstate)                                  |
+| **`dispose`**                        | Dispose of the Composite                                                                                          | `() => void`                                                                     |
+| **`sendMessage`**                    | Send a message in the thread.                                                                                     | `(content: string) => Promise<void>`                                             |
+| **`sendReadReceipt`**                | Send a read receipt for a message.                                                                                | `(chatMessageId: string) => Promise<void>`                                       |
+| **`sendTypingIndicator`**            | Send typing indicator in the thread.                                                                              | `() => Promise<void>`                                                            |
+| **`removeParticipant`**              | Remove a participant in the thread.                                                                               | `(userId: string) => Promise<void>`                                              |
+| **`setTopic`**                       | Set the topic for the thread.                                                                                     | `(topicName: string) => Promise<void>`                                           |
+| **`updateMessage`**                  | Update a message content.                                                                                         | `(messageId: string, content: string) => Promise<void>`                          |
+| **`deleteMessage`**                  | Delete a message in the thread.                                                                                   | `(messageId: string) => Promise<void>`                                           |
+| **`loadPreviousChatMessages`**       | Load more previous messages in the chat thread history.                                                           | `(messagesToLoad: number) => Promise<boolean>`                                   |
+| **`downloadAttachments`** **(Beta)** | Download a list of attachments. Add your download logic and authentication here as needed.                        | `(options: { attachmentUrls: string[] }) => Promise<AttachmentDownloadResult[]>` |
```

Comment:
Just a side not. Do we need a link explain what the `AttachmentDownloadResult` type is. Cause it looks like we do something similar for `ChatAdapterState`